changes/rate-limit-device-authed

```diff
@@ -0,0 +1 @@
+* Rate limit device authenticated endpoints when there's a failure
```

Comment:
Will review today, one concern though:
Fleet Desktop starts with menu item "Initializing..." and then runs a loop where it `GET /api/latest/fleet/device/{token}/policies` with the device token until the response is 200. (There's some latency until Fleet Server ingests the token.) Have you checked if this rate limiting on error will affect the UX at install time?

Reply:
I believe that if we allow 720 per-hour it should be a-ok.

Reply:
But maybe that's a lot since, if both Orbit+Desktop and Server are online, then things should work after ~15 seconds (from local tests).